man/faq/developer/reference-compatibility.Rmd

```diff
@@ -0,0 +1,42 @@
+
+vctrs provides a framework for working with vector classes in a generic way. However, it implements several compatibility fallbacks to base R methods. In this reference you will find how vctrs tries to be compatible with your vector class, and what base methods you need to implement for compatibility.
+
+
+## The vctrs primitives
+
+Most functions in vctrs are aggregate operations: they call other vctrs functions which themselves call other vctrs functions. The dependencies of a vctrs functions are listed in the Dependencies section of its documentation page. Take a look at [vec_count()] for an example.
+
+These dependencies form a tree whose roots are the four vctrs primitives.
```

Comment:
Could you draw the tree for `vec_count()` here? (Or if you spell it out for me, I can draw it in omnigraffle)